README.md

```diff
@@ -1,22 +1,35 @@
 # OpenFeature SDK for NodeJS
 
-![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
-![Alpha](https://img.shields.io/badge/alpha-release-red)
+[![a](https://img.shields.io/badge/slack-%40cncf%2Fopenfeature-brightgreen?style=flat&logo=slack)](https://cloud-native.slack.com/archives/C0344AANLA1)
+[![Project Status: WIP – Initial development is in progress, but there has not yet been a stable, usable release suitable for the public.](https://www.repostatus.org/badges/latest/wip.svg)]()
 [![codecov](https://codecov.io/gh/open-feature/node-sdk/branch/main/graph/badge.svg?token=3DC5XOEHMY)](https://codecov.io/gh/open-feature/node-sdk)
+[![npm version](https://badge.fury.io/js/@openfeature%2Fnodejs-sdk.svg)](https://badge.fury.io/js/@openfeature%2Fnodejs-sdk)
 [![Known Vulnerabilities](https://snyk.io/test/github/open-feature/node-sdk/badge.svg)](https://snyk.io/test/github/open-feature/node-sdk)
 
-## Alpha Checklist
+## Installation
 
-- [x] spec compliant
-- [x] contains test suite which verifies behavior consistent with spec
-- [x] contains test suite with reasonable coverage
-- [x] automated publishing
-- [ ] comprehensive readme
+```shell
+npm install @openfeature/nodejs-sdk
+```
 
-## Installation
+or
 
+```shell
+yarn add @openfeature/nodejs-sdk
 ```
-$ npm install @openfeature/nodejs-sdk   // installs the latest version
+
+## Usage
+
+```typescript
+import { OpenFeature } from '@openfeature/nodejs-sdk';
+
+OpenFeature.setProvider();
+
+const client = OpenFeature.getClient();
+
+(async () => {
+  const value = await client.getBooleanValue('enabled-new-feature', false);
+})();
```

Comment:
I'm not sure I like adding this async IIFE. I think maybe a `then` would be better?